packages/apex-node/src/tests/asyncTests.ts

```diff
@@ -126,6 +126,10 @@ export class AsyncTests {
       throw new Error(nls.localize('invalidTestRunIdErr', testRunId));
     }
 
+    if (testQueueResult === null) {
```

Comment:
This change is TBD..

Reply:
This change was removed, it was addressing a separate issue but I opened a different PR to address it #200